### PR TITLE
Update test/imports.ts

### PR DESCRIPTION
**Pull request by adidahiya**

### CODE REVIEW

**Comment by giladgray** [comment]
ok but why this one file?

File: test/imports.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+/**
+ * @license Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```
Comment (giladgray) [inline test/imports.ts]: planning on changing every other file too? 
why is this an `@license` when all the other files are normal comments?

File: test/imports.ts [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
  */
 
-/*
-Quick explanation of what's going on here:
-
-This file imports every Blueprint package with JS exports to ensure that the typings can be consumed by various
-TypeScript project configurations. A Gulp task at the bottom of `gulp/typescript.js` attempts to compile this file
-in TypeScript 2.0 with `--strictNullChecks`.
+/**
+ * @fileoverview This file imports every Blueprint package with JS exports to ensure that the typings can be
```
Comment (giladgray) [inline test/imports.ts]: does this pragma get used anywhere? does any editor understand what to do with it?

**Comment by adidahiya** [comment]
just browsing github and editing files w/out cloning right now, cleaning up low lift things. both of these are JSDoc fields

**Comment by giladgray** [comment]
i know they're JSDoc fields, but we don't consume the documentation from this file outside of the file. so it's kind of like, why bother? introducing an inconsistency with every other file.

**Comment by adidahiya** [comment]
`@fileoverview` is for humans -- it's a standard place to look

**Comment by giladgray** [comment]
i figured the block comment with full sentences at the top of the file would be enough to suggest a file overview

**Pull request merged**